core/server/data/migrations/versions/2.28/1-add-db-backup-content-permission.js

```diff
@@ -0,0 +1,61 @@
+const logging = require('../../../../lib/common/logging');
+const models = require('../../../../models');
+const utils = require('../../../schema/fixtures/utils');
+
+const fixtureBackupContentPerm = utils.findModelFixtureEntry('Permission', {
+    object_type: 'db',
+    action_type: 'backupContent'
+});
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options) {
+        try {
+            const existingBackupContentPerm = await models.Permission.findOne(
+                fixtureBackupContentPerm,
+                options
+            );
+
+            if (existingBackupContentPerm) {
+                return logging.warn('Issue adding db.backupContent (already exists)');
+            }
+
+            const result = await utils.addFixturesForModel({
+                name: 'Permission',
+                entries: [fixtureBackupContentPerm]
+            }, options);
+
+            const success = result.done === result.expected;
+
+            if (!success) {
```

Comment:
This is fucking so much more readable than the `_private.printResult` used before.. can just read through without jumping from method to method :)

Reply:
:tada: I've been playing around a lot with writing migrations, trying to make them as simple as possible - glad you like it :dancer: